packages/terra-navigation-side-menu/src/terra-dev-site/test/navigation-side-menu/NavigationSideMenuDefaultWithToolbar.test.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Base from 'terra-base';
+import NavigationSideMenu from '../../../NavigationSideMenu';
+
+class NavigationSideMenuDefault extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleOnChange = this.handleOnChange.bind(this);
+    this.resetMenuState = this.resetMenuState.bind(this);
+    this.fakeRoutingBack = this.fakeRoutingBack.bind(this);
+
+    this.state = { selectedMenuKey: 'menu', selectedChildKey: undefined };
+  }
+
+  handleOnChange(event, changeData) {
+    this.setState({ selectedMenuKey: changeData.selectedMenuKey, selectedChildKey: changeData.selectedChildKey });
+  }
+
+  resetMenuState() {
+    this.setState({ selectedMenuKey: 'menu', selectedChildKey: undefined });
+  }
+
+  fakeRoutingBack() {
+    this.setState({ selectedMenuKey: 'fake-parent', selectedChildKey: undefined });
+  }
+
+  render() {
+    let content;
+    if (this.state.selectedMenuKey === 'fake-parent') {
+      content = (
+        <div style={{
+          height: '100%', width: '100%', position: 'relative', backgroundColor: 'pink',
+        }}
+        >
+          <button type="button" onClick={this.resetMenuState}>
+            Child Route
+          </button>
+          <p>Parent Route</p>
+        </div>
+      );
+    } else {
+      content = (
+        <NavigationSideMenu
+          id="test-menu"
+          menuItems={[
+            { key: 'menu', text: 'Menu', childKeys: ['submenu1', 'submenu2', 'submenu3', 'submenu4'] },
+            {
```

Comment:
nit, want to make this object a single line like the others?

Reply:
this was just copy-pasted.. probably from auto lint change